src/Appwrite/Platform/Workers/StatsUsageDump.php

```diff
@@ -48,6 +48,7 @@ class StatsUsageDump extends Action
         METRIC_BUILDS => true,
         METRIC_COLLECTIONS => true,
         METRIC_DOCUMENTS => true,
+        METRIC_DATABASES_STORAGE => true,
```

Comment:
this only skips it from writing to the logs DB no? 🤔 

![Screenshot 2025-03-05 at 11 39 44 AM](https://github.com/user-attachments/assets/2d606d20-f5d0-472a-9a95-f7c50bbf3ca6)

But database storage calculation happens before that 
![Screenshot 2025-03-05 at 11 39 54 AM](https://github.com/user-attachments/assets/fd57c81e-2e00-46be-893b-a0dc2a188f68)

We also need to stop writing it to the projectsDB since it will be written by the StatsResoursces worker?

Reply:
Yes, that's being worked on separately 
https://github.com/appwrite/appwrite/pull/9444